sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -153,12 +153,7 @@ public void process(ProcessContext c) {}
 
   private class IdentityChildWithoutOverride extends IdentityParent {}
 
-  private class IdentityChildWithOverride extends IdentityParent {
-    @Override
-    public void process(DoFn<String, String>.ProcessContext c) {
-      super.process(c);
-    }
-  }
+  private class IdentityChildWithOverride extends IdentityParent {}
```

Comment:
I think we want to keep this because this test is to make sure we select the correct method to call when dealing with byte code generation.